cassandra/src/main/java/se/tre/freki/storage/cassandra/CassandraStore.java

```diff
@@ -415,14 +415,14 @@ public Boolean apply(final ResultSet update) {
 
         final Row row = result.get();
 
-        if (Strings.isNullOrEmpty(row.getString("meta_description"))) {
+        if (Strings.isNullOrEmpty(row.getString("description"))) {
```

Comment:
the row should be missing here if the `LabelMeta` is to be considered missing so this if clause can be removed.